source/style.css

```
@@ -473,3 +472,8 @@ body.visualization-in-focus #vertical-align {
 #controls-pane.open > .toggle:hover > svg {
 	transform: rotate(210deg);
 }
+
+#controls-pane #controls {
+	overflow-y: scroll;
```

```suggestion
	overflow-y: auto;
```
This will hide the scrollbar when the user can't scroll. I think that looks prettier :)